NetQuest OMX Pack + Integration (#37319)
demisto/content: Packs/NetQuestOMX/README.md

NetQuest’s products are high-capacity service nodes that help security teams access and analyze network traffic.

> Powerful packet & flow processing features assist security tools in detecting and mitigating security threats as cost effectively as possible. 

Reviewer: ```suggestion
Powerful packet and flow processing features assist security tools in detecting and mitigating security threats as cost effectively as possible. 
```

Key product capabilities include:

> Streaming Network Sensor (SNS) - converts traffic to standard or L7-enriched IPFIX flow data
> Packet Services Broker (PSB) - classifies & optimizes packet flows sent to security tools
> WAN Signals Probe (WSP) - auto-discovers & targets traffic in OTN/SDH/SONET links

Reviewer: ```suggestion
- Streaming Network Sensor (SNS) - converts traffic to standard or L7-enriched IPFIX flow data.
- Packet Services Broker (PSB) - classifies and optimizes packet flows sent to security tools.
- WAN Signals Probe (WSP) - auto-discovers and targets traffic in OTN/SDH/SONET links.
```

## What does this pack do?
Using the NetQuest OMX integration you can:

> - Uploads a .txt file with address list to the appliance. The appliance temporarily stores the file until it is saved to the Library and replaces any previously loaded list file.

Reviewer: ```suggestion
- Upload a .txt file with address list to the appliance. The appliance temporarily stores the file until it is saved to the Library and replaces any previously loaded list file.
```

> - Optimize Updated Address List - If the traffic elements are IP addresses, the integration should optimize the list by compressing IP addresses into CIDR groups.

Reviewer: ```suggestion
- Optimize updated address list - If the traffic elements are IP addresses, the integration will optimize the list by compressing IP addresses into CIDR groups.
```

> - Optimize updated address list - If the traffic elements are IP addresses, the integration will optimize the list by compressing IP addresses into CIDR groups.

Reviewer: ```suggestion
- Optimize updated address list. If the traffic elements are IP addresses, the integration will optimize the list by compressing IP addresses into CIDR groups.
```

- Create a new address list.

> - Changing the name of an exists list.  

Reviewer: ```suggestion
- Change the name of an existing address list.  
```

- Delete an exists list.

> - Fetch events every period of time. Available just in XSIAM.

Reviewer: ```suggestion
- Fetch events every period of time. Available only in Cortex XSIAM.
```
